app/Document/Paragraph.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Document;
+
+use App\Document\Sentence;
+
+class Paragraph
+{
+    public $content;
+    public $number;
+
+    public function __construct($content, $number = null)
+    {
+        $this->content = $content;
+        $this->number = $number;
+    }
+
+    public function get()
+    {
+        return $this->paragraphs();
+    }
+
+    public function paragraphs()
+    {
+        $paragraphs = collect(explode("\n\n", $this->content))
```

Comment:
What happens if the paragraph is following APA format and it starts with a tab instead of two new lines?